### PR TITLE
Community ideas and archive

### PR DESCRIPTION
**Pull request by sims1253**

### CODE REVIEW

File: community/community_ideas.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+Newcomer Experience
+-------------------
+
+- How to attract more newcomers
+	- How to cope with the increasing amounts of newcomers
+- How to teach and help newcomers scalable so we can keep working on coala.
+	- Talk to Docs team
+- How to improve newcomer -> dev/maintainer ratio
+- How to increase amount of feedback we get
```
Comment (sils) [inline community/community_ideas.md:26]: we already get quite some feedback FWIW we should do more with it as well

File: community/community_ideas.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+- Sponsor hunt
+
+
+Merch
+-----
+
+- Shirts
+- Stickers
+- Flyers
+- Banners
```
Comment (sils) [inline community/community_ideas.md:84]: driiiiinks!

File: community/community_ideas.md (hunk 1 above)
Comment (sims1253) [reply community/community_ideas.md:26]: added

File: community/community_ideas.md (hunk 2 above)
Comment (sims1253) [reply community/community_ideas.md:84]: added

**Pull request merged**